components/casemap/src/provider/data.rs

```diff
@@ -367,7 +367,7 @@ impl CaseMapDataULE {
 /// 6. The equality invariant is satisfied
 unsafe impl ULE for CaseMapDataULE {
     fn validate_byte_slice(bytes: &[u8]) -> Result<(), UleError> {
-        let sixteens = RawBytesULE::<2>::parse_byte_slice(bytes)?;
+        let sixteens = RawBytesULE::<2>::parse_bytes(bytes)?;
```

Comment:
Thought: could be `try_from_bytes`

Reply:
I like this being `parse()` but it's worth discussing. I'd want to do that separately IMO.